daemon/stop.go

```diff
@@ -89,3 +90,21 @@ func (daemon *Daemon) StopPodWithinLock(pod *Pod) (int, string, error) {
 
 	return vmResponse.Code, vmResponse.Cause, nil
 }
+
+func (daemon *Daemon) StopContainer(container string) error {
+	pod, idx, ok := daemon.PodList.GetByContainerIdOrName(container)
+	if !ok {
+		return fmt.Errorf("can not found container %s", container)
+	}
+
```

Comment:
Transition Lock may need there.But I'll do it in the next pr which update  container status of the pod.